.gitignore

```diff
@@ -0,0 +1,86 @@
+target/
```

Comment:
Temporary files are still in the PR, .gitignore file is still not correct.

Reply:
.gitignore is correct. The reason is that /target is tracked by git. Thus, it is not ignore although it is added to .gitignore. 
Solution: remove the track with git rm -r --cached /target